docusaurus/docs-internals/editor-controls/data-controls/transform.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 2
+toc_max_heading_level: 4
+---
+
+# Transform
+
+Brizy's transform control provides users with the ability to quickly and easily adjust the position of any component within their design. This feature streamlines the editing process, allowing for precise placement and real-time adjustments to enhance the overall layout and user experience.
+
+Example of the transform:
+
+![Transform](/img/data-controls/transform.png)
+
+### Parameters
+
+| Name                      | Type                                                                                                                                                                            | Default | Description                                                                                                                                                                                                                       |
+|:--------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| :-----: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                      | `string`                                                                                                                                                                        |    -    | The identifier of the key where the inputText will save your data                                                                                                                                                                 |
+| `type`                    | `string`                                                                                                                                                                        |    -    | Type should be `"inputText"` to use this control                                                                                                                                                                                  |
```

Comment:
about "Type should be `"inputText"` to use this control"

why inputText ? set your control name